src/Stratis.Features.Diagnostic/DiagnosticSettings.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Text;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Utilities;
+using Stratis.Features.Diagnostic.PeerDiagnostic;
+
+namespace Stratis.Features.Diagnostic
+{
+    /// <summary>
+    /// Configuration related to the API interface.
+    /// </summary>
+    public class DiagnosticSettings
+    {
+        /// <summary>The default value for peers statistics collector.</summary>
+        public const bool DefaultPeersStatisticsCollectorEnabled = false;
+        
+        /// <summary>The default value for maximum peers logged events.</summary>
+        public const int DefaultMaxPeerLoggedEvents = 10;
```

Comment:
Won't this fill up quickly? Maybe make it a 100?

Reply:
my concern was just memory consumption in case of many peers. in my tests it has been enough to diagnostic the problems when I made use of this feature and there is an argument to increase it at will